ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -17,25 +17,40 @@
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
-{%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
+{%         if router_pim_sparse_mode.ipv4.anycast_rps is arista.avd.defined %}
 
 ### IP Anycast Information
 
 | IP Anycast Address | Other Rendez-vous Point Address | Register Count |
 | ------------------ | ------------------------------- | -------------- |
 {%             for anycast_rp in router_pim_sparse_mode.ipv4.anycast_rps | arista.avd.natural_sort %}
 {%                 for other_anycast_rp_address in router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses | arista.avd.natural_sort %}
-| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {% if router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is defined and router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is not none %} {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count }} | {% else %} - {% endif %}|
+| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {% if router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is arista.avd.defined %} {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count }} | {% else %} - {% endif %}|
```

Comment:
Please use arista.avd.default filter. This line can be changed to:

```
| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count | arista.avd.default('-') }}
```

Reply:
Not required but I would recommend to set a variable before and use it in the above example.

```
{% set register_count = router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count | arista.avd.default('-') %}
```
Check the indentation, it is different. Now you can use ```register_count``` in the above example. This will be easy for users to read the code.

Reply:
Done